blocks/init/src/Blocks/custom/tabs/tabs-block.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { TabsEditor } from './components/tabs-editor';
+import { checkAttr } from '@eightshift/frontend-libs/scripts/helpers';
+import manifest from './manifest.json';
+
+
+export const Tabs = (props) => {
+	const { clientId, setAttributes, attributes } = props;
+	const tabsBlock = wp.data.select('core/block-editor').getBlock(clientId);
+	let inner = [];
+	tabsBlock.innerBlocks.forEach((tab) => {
+		inner.push(tab.attributes);
+	});
+	props.inner = inner;
```

Comment:
```suggestion
	const { clientId, setAttributes, attributes } = props;
	const tabsBlock = wp.data.select('core/block-editor').getBlock(clientId);
	
	let inner = [];
	tabsBlock.innerBlocks.forEach((tab) => {
		inner.push(tab.attributes);
	});
	
	props.inner = inner;
```

The `inner` foreach could be replaced with `map` I think.